service/auth.service.js

```diff
@@ -53,7 +56,10 @@ class AuthService {
     return this.userRepository.getUserByEmail(email).then(user => {
       if (!AuthService.isUserValid(user, password)) throw Boom.unauthorized('Incorrect email or password!')
 
-      const tokenPayload = { email: user.email }
+      const tokenPayload = {
+        id: user.id,
+        email: user.email,
+      }
 
       return AuthService.encryptJwt(tokenPayload)
```

Comment:
Let's maybe create a separate function that will take `user` as parameter and return token payload?
We need such a payload in two places.